ims/einstein/ceph.py

```diff
@@ -48,7 +48,7 @@ def __init_context(self):
 
     @trace
     def __init_cluster(self):
-        cluster = rados.Rados(rados_id=self.rid, conffile=self.r_conf)
+        cluster = rados.Rados(conffile=self.r_conf)
```

Comment:
I didn't have issues in my docker container or before, @sirushtim could you reproduce this ?